appveyor.yml

```diff
@@ -13,7 +13,7 @@ configuration:
 before_build:
   - git submodule update --init --recursive
   - python get_steinberg_sdk.py
-  - cmake -G "Visual Studio 14" Smexoscope
+  - cmake -G "Visual Studio 14 2015 Win64" .
```

Comment:
Perhaps it would be cool to build both 32 and 64 bit plugins...

Btw, you have any idea about the runtime linking? is cmake defaulting to static linking? because for VST plugins this is kind of recommended. Then at least people don't need to install the redistributables.